src/LoginDialog/OidcHandler.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import { useLocation, useHistory } from 'react-router-dom';
+import { oidcLogin } from '../service';
+import { useCustomerData } from '../app-state';
+import { generateRedirectUri } from './OidcUtilities'
+
+export const OidcHandler: React.FC<any> = ()=> {
+
+    const history = useHistory()
+    const location = useLocation()
+    const { setCustomerData } = useCustomerData();
+
+    useEffect(() => {
+        async function setCustomerDataFromOidcCallback() {
```

Comment:
In an ideal world this callback would be handled on a completely independent page that doesn't load any external javascript or CSS, but it probably doesn't matter that much.

Reply:
There should not be any extraneous javascript or css being loaded except for the loading indicator on this page.  The header bar and the footer should not reload when switching pages.